environments/staging/inventory.ini.j2

```diff
@@ -71,7 +71,7 @@ hqdb1
 # celery0
 
 [hqpillow0]
-hqpillow0-staging.internal-va.commcarehq.org
+# {{ hqpillow0-staging }} hostname=hqpillow0-staging ec2=yes
 
 [pillow0]
 # {{ pillow0-staging }} hostname=pillow0-staging ec2=yes
```

Comment:
You'll want to uncomment this and then run

```
cchq staging aws-fill-inventory
```
(after the machine has been created through `cchq staging terraform apply`) to get the real IP copied to `inventory.ini`

Reply:
I just read your description more closely, so I now realize this comment wasn't that helpful, since you're planning on doing this as the following step.

Reply:
Ok cool, and that's what I should be doing. Cool
(Btw, I sent you some slacks, letting you know here because you're snoozed there but seem to be checking this)